### DISCUSSION TITLE
"build -- --out-dir build.path " out-dir with dot do not work

### DISCUSSION THREAD

**Discussion by 668168** [facebook/docusaurus#7228] [Q&A] [answered]
by nginx , default webpage path is the domain name. 
for example, create a new site named meta.com by nginx, it will gen a folder named meta.com.
i want to use docusaurus build the --out-dir to the folder named meta.com, will be failed.
if the folder name has not dot, all will be OK.

~/ws/docusaurus # npm run build -- --out-dir test.output                                                                                                        
```

[ERROR] Error: EISDIR: illegal operation on a directory, open '/root/ws/docusaurus/test.output'
[ERROR] Unable to build website for locale en.
[ERROR] Error: EISDIR: illegal operation on a directory, open '/root/ws/docusaurus/test.output'

```

**Comment by slorber** [comment, marked as answer]
Agree this is unexpected

Even our own website can't build on a folder with a dot:

```js
yarn build:website:fast --out-dir test.output
```

It looks like the presence of a dot makes some code things the output folder looks like a file (with an extension) instead of a dir.

Failure happens during the Webpack compilation phase. I guess it's both for client/server considering there are 2 errors printed.
